runtime/doc/eval.txt

```diff
@@ -1794,7 +1794,7 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 			abort		Whether the event triggered during
 					an aborting condition (e.g. |c_Esc| or
 					|c_CTRL-C| for |CmdlineLeave|).
-			chan		|channel-id| or 0 for "internal".
+			chan		|channel-id|
```

Comment:
@zeertzjq Is there any plan to add an alternative way to detect if neovim is using the builtin UI?

According to https://github.com/vimpostor/vim-tpipeline/issues/46#issuecomment-1432140242 the only way in nightly is to test for `nvim_list_uis()[0].ext_termcolors`, which feels a bit hacky and I have a hunch it will break in the future.
I think it would be important to add official API for this before the next neovim release, as otherwise users would need to add an extra condition just for neovim 0.9.

Perhaps the elements of `nvim_list_uis()` could have a bool for that, or we could add a `name` field, so that we are able to identify UIs and the internal UI would set `name` to `internal`?

Reply:
`has('gui_running')` has just been added.